File: data/data_transpiled/at_coder/abc012/B/4905082.java
public static int n ( ) {
  n = input . nextInt ( ) ;
  h = ( int ) ( n / 3600 ) ;
  m = ( int ) ( ( n - h * 3600 ) / 60 ) ;
  s = n - h * 3600 - m * 60 ;
  return s ;
}

File: data/data_original/at_coder/abc012/B/4905082.py
n=input()
n=int(n)
h=int(n/3600)
m=int((n-h*3600)/60)
s=n-h*3600-m*60
print(str(h).zfill(2),end="")
print(":",end="")
print(str(m).zfill(2),end="")
print(":",end="")
print(str(s).zfill(2))